### DISCUSSION TITLE
内存需求具体有多大呢？我给了800m以及1c的配置；运行定时任务之后网页无法打开

### DISCUSSION THREAD

**Discussion by tefuirZ** [qd-today/qd#264] [General]
docker运行的，用的乌班图的系统。只要一运行定时任务就卡死页面。在log中也没有任何提示，怀疑是性能不足导致的。请问性能最低有多少的要求吗

**Comment by acooler15** [comment]
没有做过相关测试

**Comment by a76yyyy** [comment]
是网页端卡死么,可以看看浏览器控制台报错

**Reply by a76yyyy** [reply]
如果确认是性能问题,可以考虑使用精简镜像 a76yyyy/qiandao:lite-latest
